server/publications.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable meteor/audit-argument-checks */
```

Comment:
😮

Reply:
Basically that checks for if the arguments for Meteor methods are "checked" but the docs said you should only care if you're using the Meteor checks package, which we aren't.

https://github.com/dferber90/eslint-plugin-meteor/blob/master/docs/rules/audit-argument-checks.md

Reply:
@draycfarley Then it might be more helpful to disable it in `.eslintrc.json`>

Reply:
Moved it to eslintrc in latest commit @SamChou19815